src/net/sourceforge/kolmafia/request/PingRequest.java

```diff
@@ -55,6 +55,13 @@ private boolean isSafeToRun() {
     };
   }
 
+  private static boolean canFollowRedirect(String redirectLocation) {
```

Comment:
I can come with reasons why I would have a well named function that currently returned a constant value but my decision to do so has been questioned here and elsewhere.  Do you anticipate a time when the function would be non constant?

Reply:
I initially followed the redirect, using the code I have there.

When I decided not to follow the redirect, I decided to keep the code, rather than having to rewrite it if I did come up with a redirect I wanted to follow.

I could remove the function and the currently unreachable code.

Reply:
Your call.  If there is any chance a valid opportunity to redirect occurs then it will be easy to respond.  On the other hand the generation of devs who grew up with version control systems have pointed out that if that code is ever needed again it can be recovered from the repository.

Reply:
I decided to get rid of the dead code.